packages/next/build/index.ts

```diff
@@ -208,13 +208,9 @@ export default async function build(dir: string, conf = null): Promise<void> {
   const staticCheckSema = new Sema(config.experimental.cpus, {
```

Comment:
Does `jest-worker` still need a semaphore?

Reply:
Yeah, it's probably still good to use the semaphore

Reply:
IIRC we only needed this for flying shuttle (because the bundles were so big). @timneutkens recently remarked that `cpu * 3` was actually pretty small -- can we try to remove it and see if it works on `front`?

Reply:
Like, `jest-worker` should already be handling "semaphoring" right? There's only so many workers -- I feel like it's better to defer the backoff/queuing to `jest-worker`.

Reply:
Also, we can probably remove the semaphore from the terser taskrunner as well.

Reply:
Letting `jest-worker` handle the queueing makes sense, removed the semaphore from both and appears to handle `front` fine locally